.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,67 @@
+{
+	"name": "hnp",
+	"build": {
+		"args": {
+			"ENERGYPLUS_VERSION": "9.5.0",
+			"ENERGYPLUS_INSTALL_VERSION": "9-5-0",
+			"ENERGYPLUS_SHA": "de239b2e5f"
+		}
+	},
+	"dockerFile": "../Dockerfile",
+	"context": "..",
+	"settings": {
+		"terminal.integrated.profiles.linux": {
+			"bash": {
+				"path": "bash",
+				"icon": "terminal-bash"
+			},
+			"zsh": {
+				"path": "zsh"
+			},
+			"fish": {
+				"path": "fish"
+			},
+			"tmux": {
+				"path": "tmux",
+				"icon": "terminal-tmux"
+			},
+			"pwsh": {
+				"path": "pwsh",
+				"icon": "terminal-powershell"
+			}
+		},
+		"terminal.integrated.defaultProfile.linux": "bash"
+	},
+	"extensions": [
```

Comment:
Are all the following extensions needed?

Reply:
No these were just what was being used in the Sinergym dev container, but I have removed them now.